src/subscript/config_jobs/PRTVOL2CSV

```diff
@@ -5,14 +5,16 @@ DEFAULT <DIR> share/results/volumes
 DEFAULT <OUTPUTFILENAME> simulator_volume_fipnum.csv
 
```

Comment:
Delete this empty line? Also delete empty line 2?

Reply:
Sure, done.